lib/takeaway.rb

```diff
@@ -0,0 +1,50 @@
+class Takeaway
+
+  DEFAULT_MENU = { fish: 600, chips: 200, peas: 100 }
+
+  def initialize(initial_menu = DEFAULT_MENU)
+    @menu = initial_menu
+    @selection = Hash.new(0)
+  end
+
+  def present_menu
+    return "Sorry – no items on the menu today" if menu.empty?
+    menu.map { |d, p| "#{d.to_s.capitalize}: #{pounded(p)}" }.join(', ')
+  end
+
+  def select_item(dish, quantity = 1)
+    selection[dish] = quantity
+    selection.delete(dish) if quantity == 0
+    "Selected #{quantity} #{dish.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation.